### DISCUSSION TITLE
Welcome to Metaculus Discussions!

### DISCUSSION THREAD

**Discussion by George3d6** [Metaculus/metaculus#124] [Announcements]
@SylvainChevalier do you think it's ok to move all of the metaculus platform discussions here? I.e. besides discussion on individual questions.

**Comment by SylvainChevalier** [comment]
What do you mean by platform discussions? Stuff like [this post](https://www.metaculus.com/questions/935/platform-feature-suggestions/), or discussions in slack, or internal notion docs, or..?

**Reply by George3d6** [reply]
more so referring to that post, but, in hindsight, I think this GitHub section is best left for technical discussions.
